assets/minecraft/models/item/diamond_sword.json

```diff
@@ -75,7 +75,7 @@
 		{ "predicate": {"custom_model_data": 69}, "model": "item/mob/chimokami/chimokami_run"},
 		{ "predicate": {"custom_model_data": 70}, "model": "item/mob/chimokami/chimokami_hit"},
 		{ "predicate": {"custom_model_data": 71}, "model": "item/mob/demonfish/demonfish"},
-		{ "predicate": {"custom_model_data": 72}, "model": "item/mob/demonfish/demonfish"},
```

Comment:
I saw that some of the mobs had two custom model data pathed to 1 model but others did not. I changed one of the extra paths to demonfish model and replaced it with a path to the demonfish_move model

Reply:
Yep, anything that has a double for their second model is a placeholder for an animation that needs to be done